src/dataDisplay/popover/popover.jsx

```diff
@@ -0,0 +1,140 @@
+import Classnames from 'classnames';
+import { get } from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+    UI_CLASS_NAME,
+    BEM_POPOVER,
+} from '../../global/constants';
+import makeStyles from '../../styles/makeStyles';
+import ToolTip from '../tooltip';
+
+const propTypes = {
+    /**
+     * The arrow can be enabled/disabled
+     */
+    arrow: PropTypes.bool,
+    /**
+     * Override or extend the styles applied to PersonPanel.
```

Comment:
This should say "Popver" instead of "PersonPanel" (classic copy-and-paste error)

Reply:
I will fix this in just a sec. @War777 is taking some needed rest.

Reply:
Fixed.

Reply:
Fixed in 9561193.  Resolving.

Reply:
Fixed.